src/test/scala/sigmastate/utxo/SpamSpecification.scala

```diff
@@ -32,7 +33,7 @@ class SpamSpecification extends SigmaTestingCommons {
     (1 to 1000000).foreach(_ => hf(block))
 
     val t0 = System.currentTimeMillis()
-    (1 to 4000000).foreach(_ => hf(block))
+    (1 to 6000000).foreach(_ => hf(block))
```

Comment:
Why did it required to increase the timeout? 6M hashes is quite a lot for single script verification. Also variable description still contains `we assume that verifier must finish verification of any script in less time than 3M hash calculations`

Reply:
added #510 to fix this